### ISSUE TITLE
How can we integerate it in existing react app, I have not found any docs related to it in npm registery.

### ISSUE THREAD

**Issue by rehan-focusteck** [tuanpham-dev/react-invoice-generator#37]

**Comment by ciuliene** [comment]
I was wondering the same thing...

**Comment by marsidorowicz** [comment]
i think you can copy paste, this i what i would do

**Comment by zaynekomichi** [comment]
Here s a repository if you want to contribute, ill get started with a mockup

**Comment by nishu-murmu** [comment]
> It would be a great idea to create an npm package that will allow users to use templates. If anyone is interested just let me know and we can make an open-source repository

Yeah, I was thinking of this as well, we can fork this repo to implement such functionality where we can programmatically update the field values, enter values and other tweaks.
Let me know if anyone is interested.
